src/Microsoft.AspNetCore.Mvc.RazorPages/UrlHelperExtensions.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Routing;
+
+namespace Microsoft.AspNetCore.Mvc
+{
+    /// <summary>
+    /// Razor Page specific extensions for <see cref="IUrlHelper"/>.
+    /// </summary>
+    public static class UrlHelperExtensions
+    {
+        /// <summary>
+        /// Generates a URL with an absolute path for the specified <paramref name="page"/>.
+        /// </summary>
+        /// <param name="urlHelper">The <see cref="IUrlHelper"/>.</param>
+        /// <param name="page">The page to generate the url for.</param>
+        /// <returns>The generated URL.</returns>
+        public static string Page(this IUrlHelper urlHelper, string page)
```

Comment:
We also need one of this https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.Core/UrlHelperExtensions.cs#L16

It gets the URL of the current page.